### PR TITLE
Add the AMA - Question & Question Links to Markdown Script

### PR DESCRIPTION
**Pull request by ni5arga**
This script is designed to generate a Markdown file containing questions and links from a Reddit post's comments. It specifically focuses on questions or comments answered by a specific user. 

The resulting Markdown file will contain a list of questions or comments that the specified guest has answered, with each question numbered and linked to the corresponding Reddit comment.

### CODE REVIEW

File: AMA Script/main.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+username = os.environ["REDDIT_USERNAME"]
+
+# Create a Reddit instance
+reddit = praw.Reddit(client_id=client_id,
+                     client_secret=client_secret,
+                     password=reddit_pass,
+                     user_agent=user_agent,
+                     username=username)
+
+post_url = 'https://reddit.com/r/developersIndia/' # replace with AMA post URL
```
Comment (Bhupesh-V) [inline AMA Script/main.py]: Let's convert this to a CLI argument, or ask user for adding the post url

File: AMA Script/main.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+submission = reddit.submission(url=post_url)
+submission.comments.replace_more(limit=None)
+
+guest_username = 'AMA_GUEST_USERNAME' # replace with AMA guest name
```
Comment (Bhupesh-V) [inline AMA Script/main.py]: Same as above, either CLI arg or input

**Review by Bhupesh-V — CHANGES_REQUESTED** [review]
LGTM, few changes

**Also, let's rename the script to `ama-summarizer`**

**Comment by ni5arga** [comment]
Done.

**Review by Bhupesh-V — APPROVED** [review]

**Pull request merged**